core/storage/user/gae_models.py

```diff
@@ -59,6 +63,14 @@ class UserSettingsModel(base_models.BaseModel):
     username = ndb.StringProperty(indexed=True)
     # Normalized username to use for duplicate-username queries. May be None.
     normalized_username = ndb.StringProperty(indexed=True)
+    # Display name for users.
```

Comment:
How does this differ from the username field? Choose names and write descriptions so that it is very clear.